test/converters.test.ts

```diff
@@ -406,3 +406,30 @@ test("dynamic decimal converter", async () => {
   expect(field.value).toEqual("3.141");
   expect(field.error).toEqual("Could not convert");
 });
+
+test("text string array converter", async () => {
+  const M = types.model("M", {
+    foo: types.array(types.string)
+  });
+
+  const form = new Form(M, {
+    foo: new Field(converters.textStringArray)
+  });
+
+  const o = M.create({ foo: ["A", "B", "C"] });
+
+  const state = form.state(o);
+  const field = state.field("foo");
+
+  await field.setRaw("A\nB\nC");
```

Comment:
I think a test that verifies what happens if raw is the empty string would be useful.